pkg/storage/tree/profile.proto

```diff
@@ -0,0 +1,211 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
We already have this file in a different package (`convert`), so ideally we should avoid creating another copy.

Reply:
Same with profile.pb.go

Reply:
This is something I wanted bring up with you, If I use the proto file in `convert` it gives a circular imports error. What's a good way to handle this you think?

Reply:
That's a good point. It seems the tree parser (`ParseTreeNoDict`) is in the `convert` package causing the cycle. It probably makes sense to move the parser to the `tree` package`.

Reply:
Good suggestion